packages/preactement/tests/parse.spec.ts

```diff
@@ -36,14 +35,6 @@ describe('parse', () => {
       expect(instance.text()).toEqual(testHeading);
     });
 
-    it('handles whitespace within custom element', () => {
```

Comment:
**suggestion**: Could we replace this unit test with one that ensures the whitespace is correctly maintained? That way, future changes that might cause a regression here can be avoided. It might be an idea to use the HTML structure in your issue repro 👍

Reply:
Done ✅